src/Application/Application.php

```diff
@@ -146,7 +148,9 @@ public function getResourcesPath($path = '')
      */
     public function tablePrefix()
     {
-        return $this->reference . '_';
+        $app = DB::select('select * from applications where domain = "' . \Request::server("HTTP_HOST") . '"');
```

Comment:
can you replace this part with laravel eloquent?

Reply:
@muammertop Our goal is to be able to edit without prefix. Laravel eloquent does not allow this.